pubspec.yaml

```diff
@@ -9,9 +9,7 @@ description: Transpile sass files using the "build" package.
 environment:
   sdk: '>=2.0.0-dev.9 <2.0.0'
```

Comment:
Bump this to `>=2.0.0.dev.51 <2.0.0` – corresponds to pub build/serve being removed